src/component/FruitCounters.js

```diff
@@ -0,0 +1,73 @@
+import React, {Component} from 'react';
+import FruitCounter from './FruitCounter';
+import Button from './Button';
+import '../App.css';
+
+class FruitCounters extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            bananas: 4,
+            strawberries: 0,
+            apples: 0,
+            kiwis: 2
+        };
+    }
+
+    increaseCount = (fruit) => {
+        this.setState((prevState) => ({
+            [fruit]: prevState[fruit] + 1
+        }));
+    };
+
+    decreaseCount = (fruit) => {
+        this.setState((prevState) => ({
+            [fruit]: Math.max(prevState[fruit] - 1, 0)
+        }));
+    };
```

Comment:
Ik zie liever dat er door middel van een `disabled` property de button uitgezet wordt wanneer de counter `0` is. Dan hoef je ook geen aparte functie te maken die ervoor zorgt dat de counter niet ónder de 0 gaat